src/math/FGFunction.cpp

```diff
@@ -518,12 +519,12 @@ void FGFunction::Load(Element* el, FGPropertyValue* var, FGFDMExec* fdmex,
       auto f = [](const decltype(Parameters)& p)->double {
                  return GaussianRandomNumber();
                };
-      Parameters.push_back(new aFunc<decltype(f), 0>(f, fdmex, element, Prefix, var));
+      Parameters.push_back(new aFunc<decltype(f), 0>(f, fdmex, element, Prefix, var, false));
     } else if (operation == "urandom") {
       auto f = [](const decltype(Parameters)& p)->double {
                  return -1.0 + (((double)rand()/double(RAND_MAX))*2.0);
                };
-      Parameters.push_back(new aFunc<decltype(f), 0>(f, fdmex, element, Prefix, var));
+      Parameters.push_back(new aFunc<decltype(f), 0>(f, fdmex, element, Prefix, var, false));
```

Comment:
Remove the `constForCostParams` member, it is not needed (see comment below in `FGFunction::IsConstant`)